src/LiveDevelopment/Agents/RemoteFunctions.js

```diff
@@ -824,7 +849,42 @@ function RemoteFunctions(experimental) {
     if (experimental) {
         window.document.addEventListener("keydown", onKeyDown);
     }
+    
+    var _ws = null;
+
+    function onDocumentClick(event) {
+        var element = event.target,
+            currentDataId,
+            newDataId;
+        
+        if (_ws && element && element.hasAttribute('data-brackets-id')) {
+            _ws.send(JSON.stringify({
+                type: "message",
+                message: element.getAttribute('data-brackets-id')
+            }));
+        }
+    }
 
+    window.document.addEventListener("click", onDocumentClick);
```

Comment:
I think it is better to move this call in the `onopen` callback below and
add a removeEventListener in the `onclose` callback.
Otherwise the handler will be called also when the LivePreview is terminated.